addons/mail/static/src/scss/emojis.scss

```diff
@@ -45,6 +45,13 @@ $o-mail-emoji-height: 2rem;
     bottom: 30px;
 }
 
+.o_mail_emojis_char_field {
+    .o_input{
+        min-width: 8em;
+        max-width: 90%;
+    }
+    display: inline-block;
```

Comment:
The emoji picker button didn't want to inline without this for some reason but I feel like I might have missed something,